map/framework.cpp

```diff
@@ -1132,12 +1132,13 @@ bool Framework::QueryMayBeSkipped(search::SearchParams const & params,
 
 void Framework::LoadSearchResultMetadata(search::Result & res) const
 {
-  if (res.m_metadata.m_isInitialized)
+  if (res.m_metadata.m_isInitialized || res.GetResultType() != search::Result::RESULT_FEATURE)
```

Comment:
Non feature results do not have metadata, how it can be initialized? Lets leave this field as is.